ds3-autogen-c/src/main/java/com/spectralogic/ds3autogen/c/models/StructMember.java

```diff
@@ -18,12 +18,15 @@
 public class StructMember {
     private final C_Type type;
     private final String name;
+    private final boolean isAttribute;
 
     public StructMember(
             final C_Type type,
-            final String name) {
+            final String name,
+            final boolean isAttribute) {
```

Comment:
I would keep a 2 attribute constructor that defaults `isAttribute` to false to clean up some of the test code bellow.